### PR TITLE
Add Query Rules endpoints

### PR DESCRIPTION
**Pull request by raphi**

### CODE REVIEW

**Comment by raphi** [comment]
cf. https://github.com/algolia/doc/commit/fc54d6c7d4dbcd4f1d962d591d9d22de30c3961e

**Comment by raphi** [comment]
ping @redox

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ cache: bundler
 env:
   global:
   # ALGOLIA_APPLICATION_ID
-  - secure: feDs/d2OMHhmCwVef11wwSLA5Pto+5qNr8LnijjdcnMjjIj7/HL4z1CQXkTWIyQC34NCBcW5j4HLBqHXlkFWCO7SrAQ6WACxFSVRy4OAi/XRukr88NQKx/7UD9h7ityXVJupRzw+hkw9vFWPA3AuqhDdoR1zxhIw7RB7ZY6sPqE=
+  - secure: gvZS7XrTxqUiSlZnDi4LB3G4/D9ie5WomtuRVGLGRvGebQYIGg4OoGTOpdqTvgVTOoMeACqhhZFQZMbJoKQlYcynNgNQAy6+kUd7uDB7AGThUqv1u49oEe4ftqjD31opKcb9Hdrba3+Ve/ddNfgx3EL7AgVaTQ1NozPoXOJpXyU=
   # ALGOLIA_API_KEY
-  - secure: BcxnACf8IvMYNgx/3iOaPiJ8P3QSAbdbD9wPZBlAE2DHsqgv8LFbjuJI5+o7kKJDdYKiUyOKL9Jo0WtjXPY1cZ8sQn6dtUZqrR98tHBA8gkHNuVPxNGLqO758zuQrU4bVlr/qdv3hgG/AAxih/Fj4XIcEmcZHBV+n8KmMnrSJGc=
+  - secure: Al8Ge7j4KPXKdjfEmVQBKxi2hTve8ZwDajUFoh17Mpwh674n6hkdUMcjjQ1vjrOfIU0yKwLh7E0gr+zrlqdMkvvP7fDBg4LjoUPmi/fALGCJZzUPaXilIvAUzlg8UemCrt+J8GN73kmXWW/j6XJ7WCdzFG/7DBhrCIod3FMVkDU=
```
Comment (redox) [inline .travis.yml]: Can we fallback to the regular API key now @raphi ?

**Comment by redox** [comment]
All good for merge + minor version bump + release 👍

**Pull request merged**